src/core/astish.js

```diff
@@ -1,5 +1,5 @@
 let newRule = /(?:([a-z0-9-%@]+) *:? *([^{;]+?);|([^;}{]*?) *{)|(})/gi;
-let ruleClean = /\/\*.*?\*\/|\s{2,}|\n/gm;
+let ruleClean = /\/\*+(?:.|\n)*?\*\/|\s{2,}|\n/gm;
```

Comment:
I was playing around trying to bring the added bytes down and I think by having a range with `\s` and `\S` will capture that nasty newline comment. Cause apparently `.`(dot) does not capture newlines, lol.

```suggestion
let ruleClean = /\/\*[\s\S]*?\*\/|\s{2,}|\n/gm;
```

This one adds `+2B` to master. Does it make sense?

Reply:
Good catch, this is a lot shorter :+1: :100: